docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,610 +1,127 @@
 
-```powershell
-gpupdate.exe /force
-```
+</ScopedBlock>
+- A server or virtual machine running a Windows Server operating system.
 
-## Step 5/7. Export your LDAP CA certificate
+- A host where you will run the Teleport Desktop Service.
+  <ScopedBlock scope={["oss", "enterprise"]}> This **can not** be the same instance
+  of Teleport running your Auth and/or Proxy service(s). To install Desktop Access into an existing Teleport instance running other services, see the [Manual Setup](manual-setup.mdx) guide for Desktop Access.</ScopedBlock>
 
-Teleport connects to your Domain Controller via LDAPS. This means that you must
-let Teleport know that the certificate sent by your Domain Controller during the
-initial SSL connection is trusted. If your Domain Controller's certificate is
-trusted by the system repository on the system running Teleport, you can skip
-this step.
+  See [Installation](../installation.mdx) for details.
 
-<Details title="Skipping TLS Verification">
-  If you are unable to acquire the LDAP CA certificate, you can skip
-  TLS verification by setting `insecure_skip_verify: true`. We do not recommend
-  skipping TLS verification in production environments.
-</Details>
+## Step 1/2. Install Teleport
 
-### To export a CA certificate
+### Set up the Teleport Auth and Proxy Services
 
-{/* Adapted from https://www.ibm.com/docs/it/rds/5.2.1?topic=security-exporting-certificate-from-active-directory-server */}
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
 
-1. Begin by navigating to `Start > Control Panel > Administrative Tools > Certificate Authority` to open the CA Microsoft Management Console (MMC) GUI.
-2. Right click on your CA computer and select `Properties`.
-3. From `General` tab, click `View Certificate`.
-4. Select the `Details` view and click `Copy to File`.
-5. Click `Next` in the Certificate Export Wizard, and ensure that `DER encoded binary X.509 (.CER)` is selected
-6. Select a name and location for you certificate and click through the wizard.
+## Step 2/2. Run the discovery wizard
 
-Now transfer the exported file to the system where you're running Teleport. You
-can either add this certificate to your system's trusted repository or provide
-the filepath to the `der_ca_file` configuration variable.
+In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>`mytennant.teleport.sh`</ScopedBlock>. Click on your user name at the top right and select **Manage Access**, Select "Desktop" from the main menu, then **NEXT**:
 
-## Step 6/7. Configure Teleport
+If your Windows server is already running as an Active Directory controller, skip to the next step. Otherwise, copy and paste the first command provided into a Windows PowerShell window. If your server is not already running AD Certificate Services, copy and paste the second command after the first one completes and the server restarts:
```

Comment:
```suggestion
If you already have Active Directory installed, skip to the next step. Otherwise, copy and paste the first command provided into a Windows PowerShell window. If you aren't already running AD Certificate services installed, copy and paste the second command after the first one completes and the server restarts:
```

Intentionally trying to be less vague here, because best practice is not to install AD CS on the same server as AD DS.

Reply:
```suggestion
If you already have Active Directory installed, skip to the next step. Otherwise, copy and paste the first command provided into a Windows PowerShell window. If you aren't already running AD Certificate services, copy and paste the second command after the first one completes and the server restarts:
```

Had to pick between running or installed for grammar.

Reply:
Either looks fine to me. Totally up to you.